pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/ReactiveMessageSenderBuilder.java

```diff
@@ -71,6 +72,18 @@
 	 */
 	ReactiveMessageSenderBuilder<T> maxConcurrentSenderSubscriptions(int maxConcurrentSenderSubscriptions);
 
+	/**
+	 * Sets whether an error when sending an individual message of the
+	 * {@link ReactiveMessageSender#sendMany(Publisher)} operation should stop the full
+	 * stream or not. (default: false). If true, the original error is wrapped in a
+	 * {@link ReactiveMessageSendingException} and propagated downstream. If false, a
+	 * {@link MessageSendResult} is returned with the exception put in the
+	 * {@code Exception} field and a {@code null} {@link MessageId} field.
+	 * @param stopOnError true if the send stream should stop on error, false otherwise.
+	 * @return the sender builder instance
+	 */
+	ReactiveMessageSenderBuilder<T> stopOnError(boolean stopOnError);
```

Comment:
Should this setting go in `ReactiveMessageSenderSpec` ?
Same question about `cache`, `maxInflight`, `maxConcurrentSenderSubscriptions`

Reply:
I think this is the way. The spec isn't a good place for those settings since it shouldn't be switched per message.